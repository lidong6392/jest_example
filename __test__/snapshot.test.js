import { getConfig } from '../src/snapshot.js'
test('getConfig测试', () => {
    expect(getConfig()).toMatchSnapshot()
})
// test('getConfig测试', () => {
//   const config = {server: 'https://demo.com',port: '8080'}
//   expect(getConfig()).toEqual(config)
// })
