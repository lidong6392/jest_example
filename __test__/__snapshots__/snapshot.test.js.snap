// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getConfig测试 1`] = `
Object {
  "port": "8081",
  "server": "https://demo.com",
}
`;
